assets/scss/seasons/3_harryette_mullen.scss

```diff
@@ -0,0 +1,63 @@
+.mullen {
+  padding-bottom: initial;
+  background-color: #224fc7;
```

Comment:
eventually we should move these colors into main and use them as css variables, but okay for now.